njoy_processor/Utils_Combiner.py

```diff
@@ -180,20 +196,16 @@ def BuildCombinedData(raw_njoy_data, plot = False, verbose = False):
         rx_name = "(n,{:01d}n)".format(nn)
         n_to_n_nxn_keys.append(rx_name)
     # Freegas thermal scattering
-    n_to_n_freegas_keys = ["mt221"]
+    n_to_n_freegas_keys = ["free-gas"]
```

Comment:
you are reverting to an old version of the file. do not do that. I will not be able to allow that merge request.